packages/@o3r/core/schematics/component/presenter/index.ts

```diff
@@ -83,6 +84,7 @@ function ngGenerateComponentPresenterFn(options: NgGenerateComponentSchematicsSc
     const o3rSpecPath = path.posix.join(componentDestination, `${properties.name}.spec.ts`);
     const ngStylePath = path.posix.join(componentDestination, `${properties.name}.component.scss`);
     const o3rStylePath = path.posix.join(componentDestination, `${properties.name}.style.scss`);
+    const o3rDesignTokenPath = path.posix.join(componentDestination, `${properties.name}.theme.json`);
```

Comment:
There https://github.com/AmadeusITGroup/otter/pull/1250/files#diff-e6eff8f06e5a25d2030aff90e52c72a8f88b91dc39f0495809963938b1518edeR21
You expect the file extension to be `.design-token.json`